odoo/addons/base/models/ir_cron.py

```diff
@@ -544,3 +648,18 @@ class ir_cron_trigger(models.Model):
     @api.autovacuum
     def _gc_cron_triggers(self):
         self.search([('call_at', '<', datetime.now() + relativedelta(weeks=-1))]).unlink()
+
+
+class ir_cron_progress(models.Model):
+    _name = 'ir.cron.progress'
+    _description = 'Actions Progress'
+    _rec_name = 'cron_id'
+
+    cron_id = fields.Many2one("ir.cron", required=True, index=True)
+    remaining = fields.Integer(default=0)
+    done = fields.Integer(default=0)
+    timed_out = fields.Integer(default=0)
```

Comment:
"timed out" was a boolean.
This is a counter, so please adapt the field name.
The current choice makes the code so difficult to read.

Reply:
The field has been renamed `timed_out_counter`, I suggest `timed_out_count` for consistency:

```
~/Odoo $ rg 'count = fields.Integer' | wc -l
419
~/Odoo $ rg 'counter = fields.Integer' | wc -l
5
```